cmd/nerdctl/run_windows.go

```diff
@@ -61,6 +64,17 @@ func setPlatformOptions(opts []oci.SpecOpts, cmd *cobra.Command, id string) ([]o
 		opts = append(opts, oci.WithMemoryLimit(uint64(mem64)))
 	}
 
+	hostProcess, err:= cmd.Flags().GetBool("host-process")
+	if err != nil {
+		return nil, err
+	}
+	if hostProcess {
+		opts = append(opts,
+				customopts.WithAnnotation(annotations.WindowsHostProcess, strconv.FormatBool(true)),
```

Comment:
Oh duh, I was thinking we may need to get out a hcsshim release with this annotation exported (we recently made a package that exports all of these https://github.com/microsoft/hcsshim/tree/master/pkg/annotations but no tag available with this). Forgot it exists in cri